src/projectscene/internal/projectsceneuiactions.cpp

```diff
@@ -170,10 +178,31 @@ const UiActionList ProjectSceneUiActions::m_actions = {
 ProjectSceneUiActions::ProjectSceneUiActions(std::shared_ptr<ProjectSceneActionsController> controller)
     : m_controller(controller)
 {
+    m_actions = STATIC_ACTIONS;
 }
 
 void ProjectSceneUiActions::init()
 {
+    const auto& colors = configuration()->clipColors();
+    m_actions.clear();
+    m_actions.reserve(colors.size() + STATIC_ACTIONS.size());
+
+    for (const auto& color : colors) {
+        UiAction action;
```

Comment:
previously it was necessary to register all actions with parameters... Now (if you update with the master) we can make only one static action `action://trackedit/clip/change-color`, as usual, like others